src/jdom/spec.ts

```diff
@@ -129,6 +132,14 @@ export function serviceSpecificationFromClassIdentifier(classIdentifier: number)
         || _customServiceSpecifications[classIdentifier]
 }
 
+/**
+ * Given a service specification, see if it has a test
+ * @param spec
+ */
+export function serviceTestFromServiceSpec(spec: jdspec.ServiceSpec): jdtest.ServiceTest {
+    return _serviceTests.find(t => t.service === spec.classIdentifier)
```

Comment:
best to call the field ``serviceClass`` to match other Json.

Reply:
serviceClassIdentifier instead